api/v1alpha1/openapi.yaml

```diff
@@ -2616,7 +2616,7 @@ components:
           type: string
           description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
         metadata:
-          $ref: '#/components/schemas/ObjectMeta'
+          $ref: '#/components/schemas/DeviceMetadata'
```

Comment:
cc @celdrake

Reply:
We should be aligned to the contents of this PR in https://github.com/flightctl/flightctl-ui/pull/130
I imagine we should probably wait until 0.3.0 is released to merge them both.

Reply:
Yes, but as you can see in the generated `types.gen.go` this makes DeviceMetadata an independent type, not a type composed of ObjectMeta and the Alias.